aqsconverters/aq.py

```diff
@@ -37,16 +38,20 @@ def produce_annotation(self, aq_query_type, *args, **kwargs):
         print(f"\033[33mpatched {aq_query_type} with:\033[0m", args, kwargs)    
         print("\033[33mwriting annotation here:\033[0m", aq_module_name, args, kwargs)    
 
-        run = Run(uuid1())
+        run_id = uuid1()
+        run = Run(_id=run_id,
+                  name=str(run_id))
```

Comment:
so only name? Ok, name is useful. Maybe could be useful to store something else in name which is not the same as ID? E.g. a string which consists of  `aq_query_type`, `*args`, `**kwargs` ?

Reply:
It was just a first try, mainly to see how it would end up over the graph.

But sure, more information are for sure useful, in the end it is up to us to decide what to see over the output graph.

Reply:
> It was just a first try, mainly to see how it would end up over the graph.
> 
> But sure, more information are for sure useful, in the end it is up to us to decide what to see over the output graph.

Ok. I meant two points here:

* more predicates will be needed. Though should be added with a lot of caution, with awareness of domain context.
* name (stored as title) as it is added now could be more informative. Though it's mostly cosmetic, it's good for awareness.